templates/adversarial_qa/adversarialQA/templates.yaml

```diff
@@ -1,8 +1,159 @@
 dataset: adversarial_qa
 subset: adversarialQA
 templates:
-  4b58b6af-7540-4279-a628-d03e26ac78a6: !Template
-    id: 4b58b6af-7540-4279-a628-d03e26ac78a6
-    jinja: "{{context}} \n{{question}}||| \n{{answers.text}}"
-    name: test1
-    reference: test template
+  00755780-f3c0-44b4-b159-8f3873cdb16c: !Template
+    id: 00755780-f3c0-44b4-b159-8f3873cdb16c
+    jinja: 'I want to test the ability of students to read a passage and answer questions
+      about it. Could you please come up with a good question for the passage "{{context}}"?
+      |||
+
+      {{question}}'
+    name: adversarial_qa_4
+    reference: 'Input: Context, Output: Question (generate a question)'
+    task_template: false
+  0e13b79c-850f-45e4-b0b5-f0dc352337c9: !Template
+    id: 0e13b79c-850f-45e4-b0b5-f0dc352337c9
+    jinja: "I know that the answer to \"{{question}}\" is in \"{{context}}\" somewhere.\
+      \ Can you return the offsets of the answer? \n\nSide-note for indexing/exclusive/inclusive\
+      \ range: You know what computer scientists like *wink* \n|||\n{{(answers.answer_start[0],\
```

Comment:
I'm not sure what you mean by the computer scientists comment here - can you be more specific?

Reply:
Aah, I just meant 0-indexing, and [inclusive, exclusive) range. I'll change this if it's is on the vague side.